src/components/Table/Table.jsx

```diff
@@ -176,7 +178,7 @@ class Table extends React.Component {
               </MaterialTableHeader>
               <MaterialTableBody displayRowCheckbox={false} selectable={false}>
                 {data.map((row, index) => (
-                  <MaterialTableRow key={index} style={rowStyle(highlightFn, row)}>
+                  <MaterialTableRow key={(uniqueKey && uuid.v4()) || index} style={rowStyle(highlightFn, row)}>
```

Comment:
perhaps we should use uuid.v4() always?  Index is not a good key for a table row anyway.

Reply:
Doesn't this mean every render will generate a new key?
I think we probably want to actually come up with unique keys for each row so the DOM nodes can be reused when possible.

Reply:
yea it does. I'm already looking for an alternative. I figured it out too late.